### PR TITLE
Italian language update

### PR DESCRIPTION
**Pull request by xicciodj**

### CODE REVIEW

**Comment by worproject** [comment]
There's one untranslated string: 
`GPTImgFlasherLabel=Flash GPT image with:`

File: lang/it-IT.lng [hunk 1]
```diff
@@ -99,7 +99,7 @@ PartSchemeLabel=Schema partizioni:
 ImgInstallLabel=Installa immagine con:
 GPTImgFlasherLabel=Flash GPT image with:
```
Comment (worproject) [inline lang/it-IT.lng]: This string isn't translated.

Comment (xicciodj) [reply lang/it-IT.lng]: Oops, my bad. Now it's translated.

**Comment by worproject** [comment]
Thanks! Merged.

**Pull request merged**